system/version.py

```diff
@@ -75,6 +80,49 @@ def is_dirty() -> bool:
   return dirty
 
 
+class OpenpilotMetadata(TypedDict):
+  version: str
+  release_notes: str
+  git_commit: str
+
+
+class BuildMetadata(TypedDict):
```

Comment:
why is this a typed dict at all? why not just a normal dataclass?

Reply:
i thought it made sense to leave it as a dict as thats how it's stored on disk as json, but I can make it a dataclass